### ISSUE TITLE
`autocomplete = true;` isn't working and trying to manually trigger completion produces "null"

### ISSUE THREAD

**Issue by GoldsteinE** [hrsh7th/nvim-compe#95]
```
$ nvim --version
NVIM v0.5.0-dev+1055-g8950f4e94
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/bin/cc -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/goldstein/.cache/paru/clone/neovim-git/src/build/config -I/home/goldstein/.cache/paru/clone/neovim-git/src/neovim-git/src -I/usr/include -I/home/goldstein/.cache/paru/clone/neovim-git/src/build/src/nvim/auto -I/home/goldstein/.cache/paru/clone/neovim-git/src/build/include
Compiled by goldstein@lptp

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

nvim-compe config:
```
require('compe').setup {
	debug = true;
	autocomplete = true;
	source = {
		path = true;
		nvim_lsp = true;
		nvim_lua = true;
		buffer = true;
	};
}

vimp.inoremap({'silent', 'expr'}, '<C-j>', 'compe#complete()')
vimp.inoremap({'silent', 'expr'}, '<CR>', "compe#confirm('<CR>')")
vimp.inoremap({'silent', 'expr'}, '<C-e>', "compe#close('<C-e>')")
```

Trying to press `<C-j>` anywhere instantly puts string `null` at the cursor, regardless of file type. `:mess` is empty even though `debug = true;`

**Comment by GoldsteinE** [comment]
Using `vim.cmd [[ inoremap <silent><expr> <C-j> compe#complete() ]]` instead of vimp doesn't help either.

**Comment by GoldsteinE** [comment]
Seems like compe is disabled by default, and needs explicit enabling in config.

**Comment by hrsh7th** [comment]
Fixed in #94
